.github/workflows/build.yml

```
@@ -0,0 +1,37 @@
+name: Build and push to build branch.
+
+on:
+  push:
+    branches:
+      - trunk
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build and Push
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install all dependencies
+        run: |
+             npm install
+             yarn
+             composer install
+
+      - name: Build
+        run: yarn workspaces run build
+
+      - name: Ignore .gitignore
+        run: |
+             rm package-lock.json
+             git add public_html/wp-content/themes/pattern-directory/css/* --force
+             git add public_html/wp-content/plugins/pattern-*/build --force
```

This misses the theme's JS bundle:
```suggestion
             git add public_html/wp-content/themes/pattern-directory/css/* --force
             git add public_html/wp-content/themes/pattern-directory/build --force
             git add public_html/wp-content/plugins/pattern-*/build --force
```